engine/dlib/src/zip/miniz.h

```diff
@@ -267,7 +267,7 @@
 #endif
 
 #ifdef __cplusplus
-extern "C" {
+//extern "C" {
```

Comment:
In our previous version, I had put a namespace around it all, but I think this change is easier, making the functions private to the translation unit, i.e. the zip.cpp.
This header isn't used anywhere else.